### PR TITLE
choice field to display choice text

### PR DESCRIPTION
**Pull request by aamishbaloch**
choice field added in mobile app serializer to display choice text for operating system & deployment_mechanism

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: mobileapps/serializers.py [hunk 1]
```diff
@@ -11,11 +11,19 @@ class Meta:
 class MobileAppSerializer(serializers.ModelSerializer):
     provider_key = serializers.CharField(source="provider_key_decrypted", required=False)
     provider_secret = serializers.CharField(source="provider_secret_decrypted", required=False)
+    operating_system = serializers.ChoiceField(source="operating_system_choice_text", choices=OS_CHOICES, required=False)
```
Comment (ziafazal) [inline mobileapps/serializers.py]: I don't think we need to set `source` property ChoiceField should handler it
http://www.django-rest-framework.org/api-guide/fields/#choicefield

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch Could you please see comment about ChoiceField?

**Comment by aamishbaloch** [comment]
@ziafazal Can you review the updates?

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**